lib/po.js

```diff
@@ -76,18 +76,8 @@ PO.parse = function (data) {
     headers = headers.join('\n');
     var lines = sections.join('\n').split(/\n/);
 
-    po.headers = {
-        'Project-Id-Version': '',
-        'Report-Msgid-Bugs-To': '',
-        'POT-Creation-Date': '',
-        'PO-Revision-Date': '',
-        'Last-Translator': '',
-        'Language': '',
-        'Language-Team': '',
-        'Content-Type': '',
-        'Content-Transfer-Encoding': '',
-        'Plural-Forms': '',
-    };
```

Comment:
Can we keep this? I know it's ugly, but (if I remember correctly) some tools balk when these are missing.